source/SkiaSharp.Views.WinUI/SkiaSharp.Views.WinUI/SkiaSharp.Views.WinUI.csproj

```diff
@@ -0,0 +1,28 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
@azchohfi is there an easier way to multi-target this project with the 'UWP' WinUI file above?

Reply:
You could just multi-target, but the whole structure for SkiaSharp is already like this for every single other target, so I believe it makes sense to keep it this way.

Reply:
Oh, now I noticed what happened. @mattleibow, version 2.1.2 of MSBuild.Sdk.Extras added support for net5.0 TFMs, so you could, if you want to, merge the SkiaSharp.Views.WinUI.csproj and SkiaSharp.Views.WinUI.UWP.csproj .

Reply:
Oh, yay! I'll do that. I didn't see the shiny new update!

Reply:
Seems I am still getting errors on the restore, but this is MSBuild/NuGet issues because I don't think the restore process supports the conditionals. In my case, I had the WinUI project that depends on either SkiaSharp.UWP and SkiaSharp projects depending on which TFM was selected.

I need to actually do a full multi-targeting setup here. The reason I didn't before was due to VSMac having issues and VSWin not working properly - but this is most likely fixed since I started this when multi-targeting was hot off the press.

I'm going to keep it 2 projects for now, and then start some research into multi-targeting in a new PR. This time with new build tools and IDEs.